draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -309,398 +288,424 @@ keywords will be used for the search engine. -->
-          continues to check the saved and current path information are consistent <xref target="sec-unvalid"></xref>.
-          <list style="symbols">
-                  <t>If a sender determines either that previous parameters
-                  are not valid (due to a detected change in the path) or congestion was experienced,
-                  then the sender needs to enter the Safe Retreat Phase (see below).</t>
-               	  <t>If acknowledgments show that the data sent at the unvalidated rate was successful
-                  without inducing significant congestion to the path,
-                  then the sender is permitted to continue at the rate in
-                  the Unvalidated Phase when it enters the Normal Phase.</t>
-            </list></t>
-         </section> <!-- title="Unvalidated Phase" -->
+    <section title="Reconnaissance Phase">
+        <t> When a sender resumes transmission between the same pair of endpoints,
+        (aka thinks it uses the same path) it enters the Reconnaissance Phase.
+        The sender only enters this phase when there are saved CC parameters for the same
+        pair of endpoints and this information is currently valid (i.e., the saved parameters have
+        not expired.)
+        When a method is provided (such as the BDP_Frame), a receiver can request
+        the sender to not enter this phase.</t>
+
+        <t>In this phase, the sender transmits initial data, limited by the Initial Window,
+        and monitors its reception using received acknowledgements.
+        This phase measures the current path characteristics
+        and confirms these are consistent with the previously characterized path
+        (including experiencing a similar RTT).
+
+        <list style="symbols"> <!-- overview list of phase -->
+            <t>Reconnaissance Phase: Endpoint change:
+            If the current endpoint is not the same,
+            the sender MUST revert to using the standard CC, and enters
+            the Normal Phase. If the Endpoint Token changes
+            (i.e., the saved_endpoint_token is different from the
+            current_endpoint_token), it is assumed to represent a different network path.</t>
+
+             <t>Reconnaissance Phase: RTT change: Since the CC information
+             is directly impacted by the RTT, a significant change in the RTT
+             is a strong indication that the previously estimated BDP
+             parameters are not valid for the current path.</t>
+
+            <t>Reconnaissance Phase: Lifetime of the information: The CC information is temporal.
+            Frequent connections to the same endpoint are likely to track
+            changes, but long-term use of previous values is not appropriate.</t>
+        </list></t>
+
+        <t>When a sender confirms the path
+        characteristics
+        (i.e., current_rtt &lt; 1.2*saved_rtt and &gt; 0.8 x saved_rtt) and
```

Comment:
These numbers need to be based on data - I think 0.5<RTT<2 might be a reasonable strawman, given a use Bof DP/2 as a jump? Does this seem relevant withh actual paths?